functions/backup.bash

```diff
@@ -325,19 +240,109 @@ create_amanda_config() {
   done
   echo "OK"
 
-  echo -n "$(timestamp) [openHABian] Creating Amanda backup cronjob... "
-  {
-    echo "0 1 * * * ${backupUser} /usr/sbin/amdump ${config} &> /dev/null"; \
-    echo "0 18 * * * ${backupUser} /usr/sbin/amcheck -m ${config} &> /dev/null"
-  } > "$amandaCron"
+  if ! sed -e "s|%CONFIG|${config}|g" "${BASEDIR:-/opt/openhabian}"/includes/amdump.service-template >"${serviceTargetDir}"/amdump-${config}.service; then echo "FAILED (create Amanda ${config} backup service)"; return 1; fi
+  if ! cp "${BASEDIR:-/opt/openhabian}"/includes/amdump.timer "${serviceTargetDir}"/amdump-${config}.timer; then echo "FAILED (create Amanda ${config} timer)"; return 1; fi
+  if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
+  if ! cond_redirect systemctl enable "amdump-${config}.service" &> /dev/null; then echo "FAILED (amdump-${config} service enable)"; return 1; fi
+  if ! cond_redirect systemctl enable "amdump-${config}.timer" &> /dev/null; then echo "FAILED (amdump-${config} timer enable)"; return 1; fi
```

Comment:
Why the output redirection?

Also why not `systemctl enable --now` using just `systemctl enable` will only work on reboot, right?

Reply:
right, output redir ain't needed so I removed it

... --now would trigger an immediate dump, that we don't want

Reply:
Ah, ok.

Reply:
And no the timer triggers the service that's boot independent